examples/geometry/jpsvis/01_arena.wkt

```diff
@@ -0,0 +1 @@
87, 54.520741 64.569283, 49.521305 64.569283, 49.521748 64.269279), (49.521606 63.469234, 54.521038 63.469234, 54.521038 63.769234, 49.521606 63.769234, 49.521606 63.469234), (49.521763 62.669308, 54.517761 62.669308, 54.519695 62.969307, 49.521606 62.969307, 49.521763 62.669308), (49.52177 61.869255, 54.521038 61.869259, 54.519703 62.169376, 49.521606 62.169258, 49.52177 61.869255), (49.521778 61.069328, 54.520393 61.069332, 54.520393 61.369331, 49.523296 61.369331, 49.523296 61.369316, 49.523296 61.369325, 49.521778 61.069328), (54.521038 61.869236, 54.521038 61.869241, 54.520393 61.369331, 54.521038 61.869236), (49.521786 60.269295, 49.521355 59.769238, 49.521786 60.269291, 54.520382 60.269295, 54.520382 60.569294, 49.523289 60.569294, 49.523289 60.569279, 49.523289 60.569288, 49.521786 60.269295), (49.52179 59.469238, 54.517788 59.469242, 54.520382 59.769245, 49.521355 59.769238, 49.52179 59.469238), (49.521801 58.669315, 54.521038 58.669315, 54.519733 58.969322, 49.521606 58.969318, 49.521801 58.669315), (49.521809 57.869278, 54.521038 57.869278, 54.521038 58.169277, 49.523327 58.16927, 49.523327 58.169274, 49.521809 57.869278), (49.52174 57.053967, 54.521484 57.05397, 54.519749 57.369343, 49.523319 57.366066, 49.52174 57.053967), (49.521824 56.269299, 54.521038 56.269302, 54.521038 56.569302, 49.522987 56.569302, 49.521824 56.269299), (54.521484 57.053944, 54.521484 57.053948, 54.521038 56.569302, 54.521484 57.053944), (49.521832 55.469318, 50.49062 55.469326, 50.490692 54.969326, 49.5214 54.969315, 49.520859 54.669327, 50.490734 54.669327, 50.490807 54.169281, 49.523342 54.169281, 49.520859 53.869282, 50.490852 53.869282, 50.490925 53.369354, 49.521603 53.369354, 49.521603 53.069355, 50.490967 53.069355, 50.491043 52.55442, 49.523369 52.566021, 49.521305 52.254421, 50.491089 52.254421, 50.491158 51.76926, 49.521435 51.76926, 49.521866 51.469254, 50.491203 51.469261, 50.491276 50.968971, 50.570568 50.969334, 50.720562 51.149265, 50.765499 55.569561, 53.275494 55.569054, 53.320568 51.11924, 53.470562 50.969215, 53.550579 50.969334, 53.550549 51.469261, 54.520924 51.469131, 54.519798 51.769367, 53.550549 51.76926, 53.550552 52.26931, 54.520924 52.26918, 54.519798 52.569176, 53.550556 52.569309, 53.55056 53.069107, 54.520962 53.069107, 54.519791 53.369347, 53.55056 53.36911, 53.550564 53.869282, 54.520924 53.869156, 54.519772 54.169392, 53.550568 54.169281, 53.550571 54.669327, 54.521038 54.669327, 54.519772 54.969318, 53.550575 54.969326, 53.550579 55.469326, 54.521038 55.469326, 54.521038 55.769253, 49.523335 55.769253, 49.521832 55.469318), (49.521435 50.969318, 49.521076 50.669334, 50.491318 50.669334, 50.491276 50.968971, 49.521435 50.969318), (53.550541 50.669334, 54.520233 50.669205, 54.520271 50.969208, 53.550579 50.969334, 53.550541 50.669334), (56.320286 50.169285, 56.580273 50.169285, 57.319775 50.169289, 56.320286 50.169285), (55.721016 53.369354, 55.721172 53.069355, 55.820889 53.069218, 55.820751 53.369217, 55.721016 53.369354)))
```

Comment:
Please re-check the polygon, import fails because the polygon is not simple.

FYI: You can check for simplicity with shapely to figure out what's going on.

Additionally please move the file to `examples/geometry`